bindings/cpp/newapi/result_entry.cpp

```diff
@@ -35,18 +35,28 @@ dnet_record_info lookup_result_entry::record_info() const {
 
 	deserialize(raw_data(), response);
 
+	auto copy_checksum = [](const std::vector<unsigned char> & src, unsigned char *dst) {
```

Comment:
```suggestion
	auto copy_checksum = [](const std::vector<unsigned char> &src, unsigned char *dst) {
```

Reply:
Thanks. Fixed here and for all other comments.